app/src/main/java/org/sopt/and/ui/sign/SignInScreen.kt

```diff
@@ -0,0 +1,174 @@
+package org.sopt.and.ui.sign
+
+import android.content.Context
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.R
+import org.sopt.and.extension.noRippleClickable
+import org.sopt.and.ui.component.ServiceAccountItemRow
+import org.sopt.and.ui.component.textField.WavveCommonPasswordField
+import org.sopt.and.ui.component.textField.WavveCommonTextField
+import org.sopt.and.ui.theme.Gray3
+import org.sopt.and.ui.theme.Gray4
+import org.sopt.and.ui.theme.WavveBg
+import org.sopt.and.ui.theme.WavvePrimary
+
+@Composable
+fun SignInScreen(modifier: Modifier = Modifier,
+                 email: String,
+                 password: String,
+                 onEmailChange: (String) -> Unit,
+                 onPasswordChange: (String) -> Unit,
+                 onSignUpButtonClick: () -> Unit,
+                 onSignInButtonClick: () -> Unit) {
```

Comment:
```suggestion
  fun SignInScreen(
        modifier: Modifier = Modifier,
        email: String,
        password: String,
        onEmailChange: (String) -> Unit,
        onPasswordChange: (String) -> Unit,
        onSignUpButtonClick: () -> Unit,
        onSignInButtonClick: () -> Unit) {
```
코드 개행 요런식으로 바꿔주면 더 좋을 것 같다는 생각이 들어요 ! 가독성 측면에서 뭔가 더 좋아보이네요!

Reply:
제 리뷰에 있는 내용이긴 한데, 조장님께서 남겨주신 컴포즈 인자 컨벤션 한 번 참고하시면 좋을 것 같아요!

Reply:
감사합니다!! 컨벤션이 있었군요!